bouncer/src/repo/manipulator/repo_manipulator.cpp

```diff
@@ -826,27 +826,26 @@ const repo::manipulator::modelconvertor::ModelImportConfig *config)
 	std::string fileExt = filePathP.extension().string();
 
 	std::transform(fileExt.begin(), fileExt.end(), fileExt.begin(), ::toupper);
+	
+	repo::manipulator::modelconvertor::AbstractModelImport* modelConvertor = nullptr;
 
-	if (!repo::manipulator::modelconvertor::AssimpModelImport::isSupportedExts(fileExt) 
-		&& !(fileExt == ".BIM")
-		&& !(fileExt == ".DGN"))
+	bool useAssimpImporter = repo::manipulator::modelconvertor::AssimpModelImport::isSupportedExts(fileExt);
+	bool useIFCImporter = fileExt == ".IFC" && (!config || config->getUseIFCOpenShell());
+	bool useRepoImporter = fileExt == ".BIM";
+	bool useOdaImporter = repo::manipulator::modelconvertor::OdaModelImport::isSupportedExts(fileExt);
+
+	if (!(useAssimpImporter || useIFCImporter || useRepoImporter || useOdaImporter))
```

Comment:
Instead of doing this, change the else in 849 to check for useAssimpImporter

Then add an extra else that processes REPOERR_FILE_TYPE_NOT_SUPPORTED error